engine/runner-native/src/test/resources/Factorial.enso

```diff
@@ -0,0 +1,8 @@
+fac n =
+    facacc n v = if n <= 1 then v else @Tail_Call facacc n-1 n*v
+    
+    res = facacc n 1
+    res
+    
+main =
+    fac 6
```

Comment:
Possibly we could read this from argument or OS environment variable, to make it a bit more useful program.

Reply:
Currently it does not appear to be possible to invoke main with arguments and using env vars would mean we need to import polyglot java, which we now is problematic at this stage.